src/app-layout/visual-refresh/drawers.tsx

```diff
@@ -179,6 +180,7 @@ function DesktopTriggers() {
 
   const hasMultipleTriggers = drawersTriggerCount > 1;
   const hasSplitPanel = splitPanel && splitPanelDisplayed && splitPanelPosition === 'side';
+  const [openedFromOverflow, setOpenedFromOverflow] = React.useState(false);
```

Comment:
If you open a drawer from on the page, you would want the focus to toggle back to that trigger instead of the overflow trigger anyway, so I think this is behaving as expected

Reply:
I realized we need to add the `focusDrawerClose` to the ref object in AppLayout to give users access to the focus controls for instances such as this, so I can do that in the public PR.

Reply:
Get your point, but it seems like a coincidence that the code works this way, rather than planning all along. This is why I am leaning towards writing explicit handling for it, this will help us in the future.